packages/app/src/display/style/components/ui/_overlay.scss

```diff
@@ -123,6 +123,59 @@
     }
 }
 
+.h2h-header {
+    margin-bottom: 0.3em;
+    font-size: 1.2em;
+    text-align: center;
+}
+
+.inner-profile-box {
+    display: flex;
+    flex-direction: row;
+}
+
+.name-and-health {
+    display: flex;
+    flex-direction: column;
+    font-family: Arial, sans-serif;
+    text-transform: capitalize;
+
+    &.right {
+        justify-content: right;
+        align-items: right;
+        text-align: right;
+    }
+}
+
+.player-name {
+    font-family: Arial, sans-serif;
+    font-weight: 550;
+    text-transform: uppercase;
+
+    &.right {
+        text-align: right;
+    }
+}
+
+.buffer {
+    width: 1em;
+    height: 0.6em;
+}
+
+.h2h-stat-div {
```

Comment:
Also, all of these classes are global at the moment. It would be more maintainable if you nested them like this:

```scss
.head-to-head-stats {
    .h2h-stat-div {
        /*...*/
    }

    .h2h-info-header {
        /*...*/
    }
}
```

This means they can only be used inside `.head-to-head-stats` and therefore less chance of leaking elsewhere